lib/ASRouter.jsm

```diff
@@ -1777,6 +1780,44 @@ class _ASRouter {
     this.onMessage({ data: action, target });
   }
 
+  async loadNewTabMessages(options = {}, target) {
```

Comment:
I could make this a "Hub" probably like our other ones.

Reply:
nit: make options the last argument.